README.md

```diff
@@ -8,33 +8,33 @@ This repo contains everything needed to run a musicbrainz mirror server with sea
 
 <!-- toc -->
 
-- [Prerequisites](#prerequisites)
+* [Prerequisites](#prerequisites)
   * [Recommended hardware/VM](#recommended-hardwarevm)
   * [Required software](#required-software)
   * [External documentation](#external-documentation)
-- [Components version](#components-version)
-- [Installation](#installation)
+* [Components version](#components-version)
+* [Installation](#installation)
   * [Build Docker images](#build-docker-images)
   * [Create database](#create-database)
   * [Build materialized tables](#build-materialized-tables)
   * [Start website](#start-website)
   * [Set up search indexes](#set-up-search-indexes)
   * [Enable replication](#enable-replication)
   * [Enable live indexing](#enable-live-indexing)
-- [Advanced configuration](#advanced-configuration)
+* [Advanced configuration](#advanced-configuration)
   * [Local changes](#local-changes)
   * [Docker environment variables](#docker-environment-variables)
   * [Docker Compose overrides](#docker-compose-overrides)
-- [Test setup](#test-setup)
-- [Development setup](#development-setup)
+* [Test setup](#test-setup)
+* [Development setup](#development-setup)
   * [Local development of MusicBrainz Server](#local-development-of-musicbrainz-server)
   * [Local development of Search Index Rebuilder](#local-development-of-search-index-rebuilder)
   * [Local development of MusicBrainz Solr](#local-development-of-musicbrainz-solr)
-- [Helper scripts](#helper-scripts)
+* [Helper scripts](#helper-scripts)
   * [Recreate database](#recreate-database)
   * [Recreate database with indexed search](#recreate-database-with-indexed-search)
-- [Update](#update)
-- [Issues](#issues)
+* [Update](#update)
+* [Issues](#issues)
 
 <!-- tocstop -->
```

Comment:
This list is generated using the following command:
```sh
markdown-toc -i --maxdepth 3 README.md
```

It _is valid for the `sublist` style_ according to the rule [MD004](https://github.com/DavidAnson/markdownlint/blob/main/doc/md004.md).

I would suggest to rather add a configuration file for `markdownlint` to match the existing/generated style.

Reply:
You're right, I'm adding a config file. Thanks!

Although as you mentioned the alternation of symbols would be allowed using `sublist`, the previous order (1st level dash, 2nd level asterisk) would force all 1st level lists to use asterisks:

![image](https://user-images.githubusercontent.com/8375708/233538948-c1df184c-a319-4539-9f85-296c4426e031.png)

What I'm going to do is to keep the alternation but changing the order to 1st asterisk 2nd dash.

Reply:
The ToC can be generated to comply with this new style using [`markdown-toc`](https://github.com/jonschlinkert/markdown-toc)as follows:

```bash
markdown-toc -i --bullets='*' --bullets='-' --maxdepth 3 README.md
```